### PR TITLE
feat: add ErrorButtonItem component [desktop]

### PR DESCRIPTION
**Pull request by Mayowa-Ojo**
This PR:

- Adds the `ErrorButtonItem` component [desktop]
- Adds a new color variant (**accent-secondary**) for buttons in the `Button` component

### CODE REVIEW

File: kibbeh/src/stories/ErrorButtonItem.story.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { Story } from "@storybook/react";
+import { ErrorButtonItem, ErrorButtonItemProps } from "../ui/ErrorButtonItem";
+
+export default {
+   title: "ErrorButtonItem",
+   component: ErrorButtonItem
+}
+
+export const Main: Story<ErrorButtonItemProps> = ({
+   actionButtonText="Refresh",
```
Comment (dzmitry-vasileuski) [inline kibbeh/src/stories/ErrorButtonItem.story.tsx]: Spaces around `=` missed (lines 11-13)

File: kibbeh/src/stories/ErrorButtonItem.story.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+   return (
+      <div className="w-2/5 mx-auto">
+         <ErrorButtonItem
+            actionButtonText={actionButtonText}
+            errorMessageHeading={errorMessageHeading}
+            errorMessageText={errorMessageText}
+         />
+      </div>
+   );
+}
```
Comment (dzmitry-vasileuski) [inline kibbeh/src/stories/ErrorButtonItem.story.tsx]: EOL missed

File: kibbeh/src/ui/ErrorButtonItem.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <p className="text-sm font-medium text-primary-300">{errorMessageText}</p>
+         </div>
+         <div>
+            {!!actionButtonText &&
+               <Button size="small" color="accent-secondary" className="px-3">{actionButtonText}</Button>
+            }
+         </div>
+      </BaseSettingsItem>
+   );
+}
```
Comment (dzmitry-vasileuski) [inline kibbeh/src/ui/ErrorButtonItem.tsx]: EOL missed

Comment (Mayowa-Ojo) [reply kibbeh/src/ui/ErrorButtonItem.tsx]: fixed

**Comment by ajmnz** [comment]
Looking good, will push some minor adjustments

**Pull request merged**